PSSM_score_Peptide/main.cpp

```diff
@@ -514,11 +518,16 @@ int assignPvalueToPSSMaRRAY(int argc, char *argv[])
 		}
 		if (place == -1) place = 0; //so that we get p value = 1 in this case.
 		//cout << "place = " << place << endl;
-		double p_Value = (numberOfRandomPSSM - place +0.0) / numberOfRandomPSSM;
-		if (useFactor & Seq_array.size() > 0) {
-			float factor = float(1000000) / float(Seq_array.size());
+		double p_Value = (numberOfRandomPSSM - place +0.0) / numberOfRandomPSSM;		
+		if (useFactor) {
+			int numberOfSeq = 0;
```

Comment:
This can be expensive and can be "calculated" while reading and returned in readFileToSeq_array